config/static.c

```diff
@@ -246,9 +246,12 @@ config_gatekeeper(void)
 	int ret;
 	char lua_entry_path[128];
 	lua_State *lua_state;
+	const char *base = lua_base_dir == NULL ? LUA_BASE_DIR : lua_base_dir;
+	const char *file = gatekeeper_config_file == NULL ?
+		GATEKEEPER_CONFIG_FILE : gatekeeper_config_file;
```

Comment:
Thanks to the default values, we can use variables `lua_base_dir` and `gatekeeper_config_file`  directly.